pages/overview/contributing.md

```diff
@@ -82,13 +82,35 @@ required for the pull request to be merged into main branch.
 
 ### Typical flow for introducing changes
 
-Usually development starts with cloning and checking out recent 'master'
-branch, where you will start introducing your changes.
-To create a new local branch named 'my_changes', based off 'origin/master'
+In Open CAS we use fork->PR development process, where every developer
+creates his own fork of upstream repository, and then pushes branches
+to that fork in order to create Pull Requests to the upstream repository.
+
+It's helpful to have two remotes added in local repository to have an easy
+access to both fork and upstream repositories. Example remotes configuration
+on local repository may look like this:
+
+~~~{.sh}
+$ git remote -v
+origin		git@github.com:example-user/open-cas-linux.git (fetch)
+origin		git@github.com:example-user/open-cas-linux.git (push)
+upstream	git@github.com:Open-CAS/open-cas-linux.git (fetch)
+upstream	git@github.com:Open-CAS/open-cas-linux.git (push)
+~~~
```

Comment:
Maybe before that, we can complement this instruction with a command to add a remote. I know this is pretty basic stuff but since we have a few basic commands later on, i.e. how to create commit, this one would also be beneficial. :)

Reply:
My thought was exactly the opposite - I considered removing those unnecessary details. Git is a widely known and very well documented tool, I really doubt we can create better documentation of it. Another thing is that to contribute to Open CAS you need way more git knowledge than it's described here anyway. E.g. you need `git rebase` to be able to basically address comments to your patch sets. Documenting it here is imo a big overkill.